client/components/embed-container/index.jsx

```diff
@@ -112,6 +124,24 @@ function embedTumblr( domNode ) {
 	}, 30 );
 }
 
+function embedSlideshow( domNode ) {
+	debug( 'processing slideshow for', domNode );
+
+	// set global variable required by JetpackSlideshow
+	window.jetpackSlideshowSettings = {
+		spinner: 'https://s0.wp.com/wp-content/mu-plugins/shortcodes/img/slideshow-loader.gif'
+	};
+
+	loadCSS( 'https://s0.wp.com/wp-content/mu-plugins/shortcodes/css/slideshow-shortcode.css' );
+
+	loadjQueryDependentScript( 'https://s0.wp.com/wp-content/mu-plugins/shortcodes/js/jquery.cycle.js', () => {
```

Comment:
we should check to see if our script has already loaded and just trigger the 'post-load' jquery event instead of reloading the script again